frontend/src/core/store/app.js

```diff
@@ -1,10 +1,14 @@
+import requester from '../service/requester'
+
 const state = {
   isLoading: false,
   darkMode: false
 }
 
 const actions = {
-
+  init () {
+    requester.setHeader('Authorization', localStorage.getItem('token') || '')
```

Comment:
I think we should set the token 'Authorization' header only if the localStorage token key contains something